src/sentry/snuba/metrics/datasource.py

```diff
@@ -220,15 +238,23 @@ def _build_metric_meta(
 
 def get_metrics_meta(
     projects: Sequence[Project],
-    use_case_id: UseCaseID,
+    use_case_ids: Sequence[UseCaseID],
     start: datetime | None = None,
     end: datetime | None = None,
 ) -> Sequence[MetricMeta]:
     if not projects:
         return []
 
-    stored_metrics = get_stored_metrics_of_projects(projects, use_case_id, start, end)
-    metrics_blocking_state = get_metrics_blocking_state_of_projects(projects, use_case_id)
+    has_custom_use_case_id = False
```

Comment:
```suggestion
    has_custom_use_case_id = UseCaseID.CUSTOM in use_case_ids
```

Reply:
Does it work for lists? I will check now

Reply:
It works, nice!

Reply:
ofc, python is pseudo code 😅